examples/torch/maml_trpo_ml1_metaworld_push.py

```diff
@@ -2,7 +2,7 @@
 """This is an example to train MAML-TRPO on ML1 Push environment."""
```

Comment:
nit: metaworld_ml1 not ml1_metaworld

Reply:
i will upload a commit normalizing these codebase-wide

Reply:
I renamed this file to `maml_trpo_metaworld_ml1_push.py `